Lab3.ipynb

```diff
@@ -0,0 +1,405 @@
+      "text/plain": [
+       "<PIL.JpegImagePlugin.JpegImageFile image mode=RGB size=283x45 at 0x7F40B776E790>"
+      ]
+     },
+     "execution_count": 118,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 2.2: \n",
+    "print('5 sigma probability:', str(1-stats.norm.cdf(5)))\n",
+    "Image.open('Lab3.2.jpg')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Problem 2.2: Above is the probability of 5 sigma for a normal gaussian distribution, which translates to the integral of the pdf from a value x out to infinity. Since we want to know the distance corresponding to 5 sigma, we should look for the bounding value x. Since we are looking radially in 2-d space this distance isn't too difficult to calculate. "
```

Comment:
Problem 2.2:
- using the 1-cdf for probability calculation makes sense. 
- However I think you need to use a different distribution since we are looking at the analytic pdf of a 2D Gaussian. I used stats.chi.cdf and stats.rayleigh.cdf, but if you find that it's actually a different distribution after Googling then you should use whichever distribution's cdf it is. Then I just inverted it using the stats.norm.ppf like we normally do.
- I am unsure of where d=5.489 came from. Did you use WolframAlpha or hand calculate it? I don't see if you solved it in the Python notebook. 
- I also think if you use the analytic pdf of the two Gaussians, you don't have to do that step. I sort of followed what we did in lab 2 (when we inverted the last problem to find necessary signal values) but this time to find distance